cvat/apps/engine/clowder_api.py

```diff
@@ -0,0 +1,274 @@
+import urllib.request, json
+import requests
+import os.path
+from datetime import datetime
+from enum import Enum
+
+from cvat.apps.dataset_manager.task import export_task
+from cvat.apps.engine.models import Task
+from tempfile import TemporaryDirectory
+from cvat import __version__ as cvat_version
+from django.conf import settings
+
+COCO_FORMAT_NAME = 'COCO 1.0'
+ANNOTATIONS_DIR = 'annotations'
+
+class ResourseType(str, Enum):
+    COLLECTION = 'collection'
```

Comment:
Is it used somewhere?

Reply:
@Marishka17 Not now, but Clowder has this type of entity, so we decided to leave it. It may be used in future.